### DISCUSSION TITLE
Receipt cost input to accept decimal

### DISCUSSION THREAD

**Discussion by ryansurf** [TreyWW/MyFinances#328] [Ideas]
At the moment, the receipt cost input only accepts integers(4, 5, 6, etc). I think it should accept up to two decimals for accuracy 

![Capture](https://github.com/TreyWW/MyFinances/assets/94500732/fd32a8f4-89c6-484c-88d6-b10b684fcbb0)

**Comment by TreyWW** [comment]
Great idea! I’ve moved this to an issue, #329, since it’s definitely a good addition to make.

Thanks for the suggestion!
